src/pip/_internal/cli/cmdoptions.py

```diff
@@ -300,18 +300,27 @@ class PipOption(Option):
 )
 
 
+EXISTS_ACTIONS = {
+    "s": "switch",
+    "i": "ignore",
+    "w": "wipe",
+    "b": "backup",
+    "a": "abort",
+}
+
+
 def exists_action() -> Option:
     return Option(
         # Option when path already exist
         "--exists-action",
         dest="exists_action",
         type="choice",
-        choices=["s", "i", "w", "b", "a"],
+        choices=list(EXISTS_ACTIONS.keys()),
         default=[],
         action="append",
         metavar="action",
         help="Default action when a path already exists: "
-        "(s)witch, (i)gnore, (w)ipe, (b)ackup, (a)bort.",
+        ", ".join(EXISTS_ACTIONS.values()),
```

Comment:
This changes the output from:

```
(s)witch, (i)gnore, (w)ipe, (b)ackup, (a)bort
```

to...

```
switch, ignore, wipe, backup, abort
```

Given that we don't take "switch" but rather "s" as input here, I'd strongly prefer that we preserve the existing style.